grakn-client/src/main/java/ai/grakn/client/Client.java

```diff
@@ -55,7 +57,20 @@
             return reader.lines().collect(joining("\n"));
         }
     };
-    
+
+    public static void main(String[] args) throws IOException {
+        String confPath = System.getProperty("grakn.conf");
```

Comment:
Put it somewhere yes. The less duplicated hardcoded strings we have the better. It's always a pain to change one and have to hunt down the 500 others you missed.